superset/connectors/sqla/views.py

```diff
@@ -221,6 +221,51 @@ class SqlMetricInlineView(CompactCRUDMixin, SupersetModelView):
     edit_form_extra_fields = add_form_extra_fields
 
 
+appbuilder.add_view_no_menu(SqlMetricInlineView)
```

Comment:
Recently we merged a PR that moved all `add_view*` calls outside of view module scope and onto `superset/app.py`.

Reply:
Oops, _removed_.